Firebase/Messaging/FIRMessaging.m

```diff
@@ -184,24 +197,36 @@ - (void)dealloc {
 #pragma mark - Config
 
 + (void)load {
-  [[NSNotificationCenter defaultCenter] addObserver:self
-                                           selector:@selector(didReceiveConfigureSDKNotification:)
-                                               name:kFIRAppReadyToConfigureSDKNotification
-                                             object:nil];
+  [FIRApp registerAsConfigurable:self];
+  [FIRComponentContainer registerAsComponentRegistrant:self];
+}
+
++ (nonnull NSArray<FIRComponent *> *)componentsToRegister {
```

Comment:
not seeing this method is called, but I guess it's the new way of implementing a category of some kind for core init?